blog/list-users-announcement.md

```diff
@@ -0,0 +1,38 @@
+---
+title: List Users API
+description: List Users API 
+slug: list-users-announcement
+date: 2024-04-30
+authors: miparnisari
+tags: [openfga,features]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# New ListUsers API on OpenFGA
+
+Today we are launching a new API for OpenFGA: ListUsers.
+
+This API will answer the question "what users have relation X with object Y?". This will be useful, for example, in UIs that want to display the list of users that a resource has been shared with, e.g. the "share" dialog in Google Docs.
+
+## How to use it?
+
+ListUsers is available in the latest Release Candidate version of OpenFGA: [v1.5.4-rc1](https://github.com/openfga/openfga/releases/tag/v1.5.4-rc1).
+
+To be able to call this API, you must turn on this flag: `--experimentals enable-list-users`. Be sure to also check out the various configuration flags that were added to control its behavior.
+
```

Comment:
Let's mention the SDKs that currently support it with link to the versions (JS, Go, .NET for now). We can add links to the rest + the CLI when they are merged.